erts/emulator/test/jit_SUITE.erl

```diff
@@ -128,4 +150,37 @@ annotate(Config) ->
             ct:log("Did not find disassembly test for ~ts.~n~ts",
                    [Symbol, Anno])
     end.
-    
+
+named_labels(_Config) ->
+    %% Check that pretty printing of named labels is working. We do
+    %% that by loading this module in an emulator running with +JDdump
+    %% true and then checking that the produced jit_SUITE.asm contains
+    %% a label for each exported function. We also check that the
+    %% label for the non-exported function get_tmp_asm_files/0, which
+    %% is used by this test, is present.
+    Exports = proplists:get_value(exports, ?MODULE:module_info()),
+    ModName = atom_to_list(?MODULE),
+    ModulePath = filename:dirname(code:which(?MODULE)),
+    Cmd = "erl +JDdump true -noshell -pa " ++ ModulePath ++ " -eval '"
```

Comment:
I think this `Cmd` will fail on Windows because of the quoting. Use " instead.

Reply:
done